test/test_reporter.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+cd test
+
+docker-compose \
+    -f docker-compose-selenium-test.yml \
+    build
+docker-compose \
+    -f docker-compose-selenium-test.yml \
+    run --rm wikibase-selenium-test npm run "test:reporter"
```

Comment:
Where is this (`test:reporter`) defined?

Reply:
this can go!